daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DataProcessor.scala

```diff
@@ -652,81 +653,11 @@ class DataProcessor private (
 
 }
 
-class ParseResult(dp: DataProcessor, override val resultState: PState)
```

Comment:
Was this just moved? It's hard to tell if there are any changes. Reccommend moving this back to make the changes more obvious if there are any.